src/main/java/org/broadinstitute/hellbender/cmdline/ArgumentCollection.java

```diff
@@ -13,4 +13,12 @@
 public @interface ArgumentCollection {
     /** Text that appears for this group of options in text describing usage of the command line program. */
     String doc() default "";
+
+    // The arguments in this collection are dependent on (and only enabled with the presence of)
+    // another argument. This may be used alone or together with dependsOnValue.
+    String dependsOnArgument() default "";
+
+    // The arguments in this collection are dependent on (and only enabled with the presence of)
+    // another argument with this specific value.
```

Comment:
No - "dependsOnValue" only makes sense if "dependsOnArgument" is specified, since it further restricts the dependency. It means "this argument collection depends on the argument(option) specified in "dependsOnArgument", but only when its specified with the value in "dependsOnValue".  So, f you had a "--debug" option that was just a boolean switch, you might use an argument collection with "dependsOnArgument=debug", but, if your "--debug" argument is an enum, you might have an argument collection that "dependsOnArgument=debug" and "dependsOnValue=3".  Perhaps there re better names ? In jopt parlance they would be "dependsOnOption" and "dependsOnArgument".